views/replace.templ

```diff
@@ -4,6 +4,18 @@ import (
 	"fmt"
 )
 
+templ Replace(selector string, component templ.Component) {
```

Comment:
Have you merged the dev branch where this is now also available?

Reply:
@nicolasfranck done